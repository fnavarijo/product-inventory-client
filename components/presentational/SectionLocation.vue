<template>
  <div class="font-bold border-b text-gray-900 border-light-gray-400 mb-5 pb-2">
    {{ path }}
  </div>
</template>

<script lang="ts">
import Vue from 'vue';

export default Vue.extend({
  name: 'SectionLocation',
  props: {
    path: {
      type: String,
      required: true,
    },
  },
});
</script>
