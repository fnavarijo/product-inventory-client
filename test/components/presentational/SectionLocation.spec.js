import { mount } from '@vue/test-utils';

import SectionLocation from '@/components/presentational/SectionLocation.vue';

describe('SectionLocation component', () => {
  it('should display path set via props', () => {
    const wrapper = mount(SectionLocation, {
      propsData: {
        path: 'home > catalog',
      },
    });

    expect(wrapper.text()).toContain('home > catalog');
    expect(wrapper.html()).toMatchSnapshot();
  });
});
