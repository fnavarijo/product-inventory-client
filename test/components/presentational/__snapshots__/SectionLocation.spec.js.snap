// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SectionLocation component should display path set via props 1`] = `
<div class="font-bold border-b text-gray-900 border-light-gray-400 mb-5 pb-2">
  home &gt; catalog
</div>
`;
